src/App.js

```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
```

Comment:
- [ ] Take a look at the image below. I was unable to run your code correctly because of the React error. This error occurred because **`jsx`** (in this case) requires React within the file. This error can be fixed by importing React at the very top of your **`App.js`** component, like so:

   ```
   import React from 'react';
   ```
  ![rich-react](https://user-images.githubusercontent.com/36542986/148183931-2d84ea32-a3fa-494a-b119-ddb901fbb4b9.jpg)